docs/src/Features/Recompute.md

```diff
@@ -0,0 +1,264 @@
+# Recompute
+
+## Where
+
+- `spyglass.spikesorting.v0.spikesorting_recording.SpikeSortingRecording`
+- `spyglass.spikesorting.v1.recording.SpikeSortingRecording`
+
+## Why
+
+Some analysis files that are generated by Spyglass are very unlikely to be
+reaccessed. Those generated by `SpikeSortingRecording` tables were identified as
+taking up tens of terabytes of space, while very seldom accessed after their
+first generation. By finding a way to recompute these files on demand, we can
+save significant server space at the cost of an unlikely 10m of recompute time
+per file.
+
+Spyglass 0.5.5 introduces the opportunity to delete and recompute both newly
+generated files after this release, and old files that were generated before
+this release.
+
+## How
```

Comment:
I think it would be helpful to give a practical example of the recompute process. Something like:
1. Let's assume there is a recording in SpikeSortingRecording saved in an nwb file at filepath.
2. I can fetch that file from disk using `fetch_nwb` on the SpikeSortingRecording table with a recording ID.
3. Now let's assume I want to save space and delete the file until it needs to be used again.
4. If I `fetch_nwb`, now spyglass checks for that file, sees that it isn't there, isn't on Dandi or on Kachery.
5. Because of this, the spyglass recomputes the file, checking to make sure the environment is the same.

And then some information about when files get deleted (do I delete the files? does Spyglass automatically delete files? , how do I delete files).